ibis/util.py

```diff
@@ -84,6 +84,8 @@ def promote_list(val: Union[V, List[V]]) -> List[V]:
     -------
     list
     """
+    if val is None:
+        val = []
```

Comment:
I don't think we should automatically turn `None` into an empty list, since it gives `None` special meaning that it is equivalent to an empty sequence. On a practical note, IIRC there are a few places in the codebase where code assumes `None` becomes `[None]`.

Reply:
That makes sense - I was curious if there were any usages that expected `[None]` but at least no tests seemed to fail as a result...